### PR TITLE
Move medical conditions question to earlier in the flow

### PR DESCRIPTION
**Pull request by brucebolt**
This moves the medical condition eligibility question to earlier in the flow, as follows:
1. Do you live in England?
2. Have you recently had a letter from the NHS?
3. Do you have a qualifying medical condition?
4. What is your name?
5. Form flow continues as previously

Trello card: https://trello.com/c/QT7Jr2qL

### CODE REVIEW

File: app/controllers/coronavirus_form/know_nhs_number_controller.rb [hunk 1]
```diff
@@ -31,6 +31,6 @@ def submit
 private
 
   def previous_path
-    medical_conditions_path
+    name_path
```
Comment (leenagupte) [inline app/controllers/coronavirus_form/know_nhs_number_controller.rb]: I think this should be: `contact_details_path`

**Review by leenagupte — COMMENTED** [review]
I think only one back link is going to the wrong place, other than that, it looks good.

File: app/controllers/coronavirus_form/know_nhs_number_controller.rb (hunk 1 above)
Comment (brucebolt) [reply app/controllers/coronavirus_form/know_nhs_number_controller.rb]: Good spot.  We should probably have a feature test for the back links.  It has caught us out (on the business volunteering form) a few times.

**Review by leenagupte — APPROVED** [review]
👍 💯

**Pull request merged**